app/controllers/versions_controller.rb

```diff
@@ -11,4 +12,49 @@ def show
     @versioned_links = @rubygem.links(@latest_version)
     render "rubygems/show"
   end
+
+  def dependencies_show
+    params[:version_id] ? latest_version_by_slug : latest_version
```

Comment:
When would `version_id` be absent?

Reply:
For this webpage we need to find latest_Version
http://localhost:3000/gems/rails

For this we need to find_by_slug since it can have different dependencies from the latest version.
http://localhost:3000/gems/rails/versions/5.2.2